gulpfile.js

```diff
@@ -1,5 +1,5 @@
 /* jshint node: true */
-'use strict';
+'use strict'; // jshint ignore:line
```

Comment:
would prefer something more targeted. Also, do you have issues with this? I don't think I've ever seen warnings about it....

(are you using gulp for linting?)